doc/rear.8.adoc

```diff
@@ -83,6 +83,9 @@ the GNU General Public License at: http://www.gnu.org/licenses/gpl.html
 -v::
     *verbose mode* (show messages what ReaR is doing on the terminal)
 
+-n --non-interactive::
+    *non-interactive mode* (ReaR aborts with an error if user input is required)
```

Comment:
Same as above:
I think non-interactive mode does not abort
in general when user input is required,
but only when user input is required via the UserInput function
under certain timeout conditions.